spec-main/chromium-spec.ts

```diff
@@ -744,20 +744,14 @@ describe('chromium features', () => {
       expect(await w.webContents.executeJavaScript('b.location.href')).to.equal('about:blank');
     });
 
-    it('sets the window title to the specified frameName', async () => {
-      const w = new BrowserWindow({ show: false });
-      w.loadURL('about:blank');
-      w.webContents.executeJavaScript('{ b = window.open(\'\', \'hello\'); null }');
-      const [, window] = await emittedOnce(app, 'browser-window-created');
-      expect(window.getTitle()).to.equal('hello');
-    });
-
     it('does not throw an exception when the frameName is a built-in object property', async () => {
       const w = new BrowserWindow({ show: false });
       w.loadURL('about:blank');
       w.webContents.executeJavaScript('{ b = window.open(\'\', \'__proto__\'); null }');
-      const [, window] = await emittedOnce(app, 'browser-window-created');
-      expect(window.getTitle()).to.equal('__proto__');
+      const [, , frameName, , test] = await emittedOnce(w.webContents, 'new-window');
```

Comment:
this issue is caused by https://github.com/eslint/eslint/issues/9760 - given that `test` is unused here and it looks like you're console logging it i think we should just disable the rule for this line and comment above:

```js
// https://github.com/eslint/eslint/issues/9760
// eslint-disable-next-line array-bracket-spacing
const [, , frameName, ,] = await emittedOnce(w.webContents, 'new-window');
```

Reply:
I don't think we shouldn't need to disable this lint rule? The `console.log` should be removed and then this line just becomes

```suggestion
      const [, , frameName] = await emittedOnce(w.webContents, 'new-window');
```

Reply:
Oh great, I wasn't aware that the return value could work in that way, but it looks to be functioning. Thank you